js/atoms/src/Block.svelte

```diff
@@ -27,7 +38,7 @@
 	class:padded={padding}
 	class:border_focus={border_mode === "focus"}
 	class:hide-container={!explicit_call && !container}
-	style:height={typeof height === "number" ? height + "px" : undefined}
+	style:height={getHeight()}
```

Comment:
I don't believe this is reactive, e.g. if height changes this won't update. `height` should be an argument to this function to make it reactive